org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ScalaProject.scala

```diff
@@ -507,11 +518,6 @@ class ScalaProject private(val underlying: IProject) extends ClasspathManagement
     settings.javaextdirs.value = " "
     settings.classpath.value = (scalaCp.userCp ++ scalaCp.scalaLibrary.toSeq).map(_.toOSString).mkString(pathSeparator)
     scalaCp.scalaLibrary.foreach(scalaLib => settings.bootclasspath.value = scalaLib.toOSString)
-
-    logger.debug("javabootclasspath: " + settings.javabootclasspath.value)
```

Comment:
Are these logged somewhere else now? They are pretty useful debugging info when the Java version isn't what you'd expect for a given project.